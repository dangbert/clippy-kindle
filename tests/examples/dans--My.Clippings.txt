Do Androids Dream of Electric Sheep? (Dick, Philip K.)
- Your Bookmark on Location 604 | Added on Friday, November 25, 2016 12:13:59 AM


==========
﻿Do Androids Dream of Electric Sheep? (Dick, Philip K.)
- Your Highlight on Location 607-608 | Added on Friday, November 25, 2016 12:14:26 AM

The tyranny of an object, he thought. It doesn’t know I exist. Like the androids, it had no ability to appreciate the existence of another. He had never thought of this before, the similarity between an electric animal and an andy.
==========
﻿Do Androids Dream of Electric Sheep? (Dick, Philip K.)
- Your Highlight on Location 2314-2317 | Added on Sunday, November 27, 2016 3:33:55 AM

I can understand now how you suffer when you’re depressed; I always thought you liked it and I thought you could have snapped yourself out any time, if not alone, then by means of the mood organ. But when you get that depressed you don’t care. Apathy, because you’ve lost a sense of worth. It doesn’t matter whether you feel better, because if you have no worth—”
==========
﻿Fahrenheit 451: A Novel (Bradbury, Ray)
- Your Highlight on Location 760-760 | Added on Monday, December 5, 2016 1:56:30 AM

“There must be something in books, things we can’t imagine, to make a woman stay in a burning house;
==========
﻿Fahrenheit 451: A Novel (Bradbury, Ray)
- Your Bookmark on Location 1089 | Added on Thursday, December 8, 2016 2:46:37 AM


==========
Los juegos del hambre (Suzanne Collins)
- Your Highlight on Location 634-635 | Added on Sunday, February 18, 2018 9:04:19 PM

quizá
==========
The 4 Hour Workweek  
- Your Note on page 60 | Added on Friday, September 8, 2017 9:04:32 PM

Important
==========
The Iliad (Penguin Classics) (Homer)
- Your Note on Location 1484 | Added on Monday, September 18, 2017 4:14:00 PM

Just read book 1
comment on how you felt about the poem as you read it.

Choose 2 moments, discuss how they made you feel and why.
==========
Encuentro en el Ártico (Eoin Colfer)
- Your Note on Location 3084 | Added on Saturday, May 30, 2020 11:49:28 AM

To scatter
==========
Encuentro en el Ártico (Eoin Colfer)
- Your Highlight on Location 3084-3084 | Added on Saturday, May 30, 2020 11:49:28 AM

se desparramó
==========
SPQR: A History of Ancient Rome (Beard, Mary)
- Your Highlight on Location 428-432 | Added on Tuesday, June 2, 2020 4:39:49 PM

It did not take long for the opening words of Cicero’s speech given on 8 November (the First Catilinarian) to become one of the best known and instantly recognisable quotes of the Roman world: ‘Quo usque tandem abutere, Catilina, patientia nostra?’ (‘How long, Catiline, will you go on abusing our patience?’); and it was closely followed, a few lines later in the written text, by the snappy, and still much repeated, slogan ‘O tempora, o mores’ (‘O what a world we live in!’, or, literally, ‘O the times, O the customs!’).
==========
SPQR: A History of Ancient Rome (Beard, Mary)
- Your Note on Location 432 | Added on Tuesday, June 2, 2020 4:40:33 PM

This is a famous quote that's still quoted/played on today.
==========
Siddhartha (Modern Library Classics) (Hesse, Hermann)
- Your Bookmark on Location 1596 | Added on Wednesday, June 10, 2020 10:41:15 AM


==========
