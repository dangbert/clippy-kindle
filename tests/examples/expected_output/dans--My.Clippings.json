[
  {
    "title": "Do Androids Dream of Electric Sheep?",
    "author": "Dick, Philip K.",
    "dateStart": "November 25, 2016 00:13:59",
    "dateEnd": "November 25, 2016 00:13:59",
    "items": [
      {
        "type": "bookmark",
        "loc": 604,
        "locType": "location",
        "dateStr": "November 25, 2016 00:13:59"
      }
    ]
  },
  {
    "title": "\ufeffDo Androids Dream of Electric Sheep?",
    "author": "Dick, Philip K.",
    "dateStart": "November 25, 2016 00:14:26",
    "dateEnd": "November 27, 2016 03:33:55",
    "items": [
      {
        "type": "highlight",
        "loc": 607,
        "locEnd": 608,
        "locType": "location",
        "dateStr": "November 25, 2016 00:14:26",
        "content": "The tyranny of an object, he thought. It doesn\u2019t know I exist. Like the androids, it had no ability to appreciate the existence of another. He had never thought of this before, the similarity between an electric animal and an andy."
      },
      {
        "type": "highlight",
        "loc": 2314,
        "locEnd": 2317,
        "locType": "location",
        "dateStr": "November 27, 2016 03:33:55",
        "content": "I can understand now how you suffer when you\u2019re depressed; I always thought you liked it and I thought you could have snapped yourself out any time, if not alone, then by means of the mood organ. But when you get that depressed you don\u2019t care. Apathy, because you\u2019ve lost a sense of worth. It doesn\u2019t matter whether you feel better, because if you have no worth\u2014\u201d"
      }
    ]
  },
  {
    "title": "\ufeffFahrenheit 451: A Novel",
    "author": "Bradbury, Ray",
    "dateStart": "December 05, 2016 01:56:30",
    "dateEnd": "December 08, 2016 02:46:37",
    "items": [
      {
        "type": "highlight",
        "loc": 760,
        "locEnd": 760,
        "locType": "location",
        "dateStr": "December 05, 2016 01:56:30",
        "content": "\u201cThere must be something in books, things we can\u2019t imagine, to make a woman stay in a burning house;"
      },
      {
        "type": "bookmark",
        "loc": 1089,
        "locType": "location",
        "dateStr": "December 08, 2016 02:46:37"
      }
    ]
  },
  {
    "title": "Los juegos del hambre",
    "author": "Suzanne Collins",
    "dateStart": "February 18, 2018 21:04:19",
    "dateEnd": "February 18, 2018 21:04:19",
    "items": [
      {
        "type": "highlight",
        "loc": 634,
        "locEnd": 635,
        "locType": "location",
        "dateStr": "February 18, 2018 21:04:19",
        "content": "quiz\u00e1"
      }
    ]
  },
  {
    "title": "The 4 Hour Workweek",
    "author": "",
    "dateStart": "September 08, 2017 21:04:32",
    "dateEnd": "September 08, 2017 21:04:32",
    "items": [
      {
        "type": "note",
        "loc": 60,
        "locType": "page",
        "dateStr": "September 08, 2017 21:04:32",
        "content": "Important"
      }
    ]
  },
  {
    "title": "The Iliad (Penguin Classics)",
    "author": "Homer",
    "dateStart": "September 18, 2017 16:14:00",
    "dateEnd": "September 18, 2017 16:14:00",
    "items": [
      {
        "type": "note",
        "loc": 1484,
        "locType": "location",
        "dateStr": "September 18, 2017 16:14:00",
        "content": "Just read book 1\ncomment on how you felt about the poem as you read it.\n\nChoose 2 moments, discuss how they made you feel and why."
      }
    ]
  },
  {
    "title": "Encuentro en el \u00c1rtico",
    "author": "Eoin Colfer",
    "dateStart": "May 30, 2020 11:49:28",
    "dateEnd": "May 30, 2020 11:49:28",
    "items": [
      {
        "type": "highlight",
        "loc": 3084,
        "locEnd": 3084,
        "locType": "location",
        "dateStr": "May 30, 2020 11:49:28",
        "content": "se desparram\u00f3"
      },
      {
        "type": "note",
        "loc": 3084,
        "locType": "location",
        "dateStr": "May 30, 2020 11:49:28",
        "content": "To scatter"
      }
    ]
  },
  {
    "title": "SPQR: A History of Ancient Rome",
    "author": "Beard, Mary",
    "dateStart": "June 02, 2020 16:39:49",
    "dateEnd": "June 02, 2020 16:40:33",
    "items": [
      {
        "type": "highlight",
        "loc": 428,
        "locEnd": 432,
        "locType": "location",
        "dateStr": "June 02, 2020 16:39:49",
        "content": "It did not take long for the opening words of Cicero\u2019s speech given on 8 November (the First Catilinarian) to become one of the best known and instantly recognisable quotes of the Roman world: \u2018Quo usque tandem abutere, Catilina, patientia nostra?\u2019 (\u2018How long, Catiline, will you go on abusing our patience?\u2019); and it was closely followed, a few lines later in the written text, by the snappy, and still much repeated, slogan \u2018O tempora, o mores\u2019 (\u2018O what a world we live in!\u2019, or, literally, \u2018O the times, O the customs!\u2019)."
      },
      {
        "type": "note",
        "loc": 432,
        "locType": "location",
        "dateStr": "June 02, 2020 16:40:33",
        "content": "This is a famous quote that's still quoted/played on today."
      }
    ]
  },
  {
    "title": "Siddhartha (Modern Library Classics)",
    "author": "Hesse, Hermann",
    "dateStart": "June 10, 2020 10:41:15",
    "dateEnd": "June 10, 2020 10:41:15",
    "items": [
      {
        "type": "bookmark",
        "loc": 1596,
        "locType": "location",
        "dateStr": "June 10, 2020 10:41:15"
      }
    ]
  }
]